src/TeleopMode.cpp

```diff
@@ -141,17 +158,106 @@ void Teleop::TeleopPeriodic() {
 
     m_wristModeSwitchPrevState = wristModeSwitch;
 
-    if (m_wristControlMode == WristControlMode::OpenLoop) {
+    /*if (m_wristControlMode == WristControlMode::OpenLoop) {
         DBStringPrintf(DBStringPos::DB_LINE2, "OPEN LOOP WRIST");
     }
     else {
         DBStringPrintf(DBStringPos::DB_LINE2, "closed loop wrist");
-    }
+    }*/
 }
 
 void Teleop::TeleopStop() {
 }
 
+void Teleop::HandlePoofsJoystick(uint32_t port, uint32_t button,
```

Comment:
Do we want to have a different handler call for each joystick?  The answer might be yes (I'll trust your judgement) but I want to make sure you've thought through the pro's and cons.  The other option would be for them all to call the same `HandleJoystick` function